BDArmory/UI/BDTISetup.cs

```diff
@@ -35,6 +35,11 @@ class BDTISetup : MonoBehaviour
         private float updateList = 0;
         private bool maySavethisInstance = false;
 
+        // Opacity Settings
+        internal const float textOpacity = 1f;
+        internal const float iconOpacity = 0.5f;
+        internal const float localToGlobalOpacity = 0.02f; // Conversion between BDATeamIcons.Opacity and BDTISettings.OPACITY
```

Comment:
Is there a reason to have this separately instead of just having `BDTISettings.OPACITY` in the range 0—1 (with `textOpacity = 2f` and `iconOpacity = 1f`)?
Also, I presume the opacity gets clamped if you set it > 1?